x-pack/metricbeat/module/aws/_meta/fields.yml

```diff
@@ -30,3 +30,27 @@
           object_type_mapping_type: "*"
           description: >
             Metrics that returned from Cloudwatch API query.
+    - name: host
+      type: group
+      fields:
+        - name: cpu.pct
+          type: scaled_float
+          description: Percent CPU used. This value is normalized by the number of CPU cores and it ranges from 0 to 1.
+        - name: network.in.bytes
+          type: scaled_float
+          description: The number of bytes received on all network interfaces by the host in a given period of time.
+        - name: network.out.bytes
+          type: scaled_float
+          description: The number of bytes sent out on all network interfaces by the host in a given period of time.
+        - name: network.in.packets
+          type: scaled_float
+          description: The number of packets received on all network interfaces by the host in a given period of time.
+        - name: network.out.packets
+          type: scaled_float
+          description: The number of packets sent out on all network interfaces by the host in a given period of time.
+        - name: disk.read.bytes
+          type: scaled_float
+          description: The total number of bytes read successfully in a given period of time.
+        - name: disk.write.bytes
+          type: scaled_float
+          description: The total number of bytes write successfully in a given period of time.
```

Comment:
We should move these fields to some common file, as they are supposed to be used in more places.

If it is kept here, it should be moved before the `aws` group. Metrics of metricsets are appended inside the last group of this file, this is probably causing the documented metrics to be moved from `aws` to `host`.

Reply:
Done! This is moved into `libbeat/_meta/fields.common.yml`